nni/retiarii/oneshot/pytorch/base_lightning.py

```diff
@@ -45,7 +49,19 @@ def apply(m):
 
 
 class BaseOneShotLightningModule(pl.LightningModule):
+
+    _custom_replace_dict_note = """custom_replace_dict : Dict[Type[nn.Module], Callable[[nn.Module], nn.Module]], default = None
+        The custom xxxChoice replace method. Keys should be ``xxxChoice`` type.
+        Values should callable accepting an ``nn.Module`` and returning an ``nn.module``.
+        This custom replace dict will override the default replace dict of each NAS method.
+    """
+
+    _base_model_note = """base_model : pl.LightningModule
+        The evaluator in ``nni.retiarii.evaluator.lightning``. User defined model is wrapped by base_model, and base_model will
+        be wrapped by this model.
```

Comment:
a little confusing, it is not clear "base_model" is a model or evaluator.

base_model is a pl.LightningModule, why it must in ``nni.retiarii.evaluator.lightning``?

It is called base_model because it contains user defined model, but it is also an evaluator...